votor/src/common.rs

```diff
@@ -36,37 +36,40 @@ pub const fn conflicting_types(vote_type: VoteType) -> &'static [VoteType] {
 ///
 /// Must be in sync with `vote_to_certificate_ids`
 pub const fn certificate_limits_and_vote_types(
-    cert_type: &Certificate,
+    cert_type: &CertificateType,
 ) -> (f64, &'static [VoteType]) {
     match cert_type {
-        Certificate::Notarize(_, _) => (0.6, &[VoteType::Notarize]),
-        Certificate::NotarizeFallback(_, _) => {
+        CertificateType::Notarize(_, _) => (0.6, &[VoteType::Notarize]),
+        CertificateType::NotarizeFallback(_, _) => {
             (0.6, &[VoteType::Notarize, VoteType::NotarizeFallback])
         }
-        Certificate::FinalizeFast(_, _) => (0.8, &[VoteType::Notarize]),
-        Certificate::Finalize(_) => (0.6, &[VoteType::Finalize]),
-        Certificate::Skip(_) => (0.6, &[VoteType::Skip, VoteType::SkipFallback]),
-        Certificate::Genesis(_, _) => (GENESIS_VOTE_THRESHOLD, &[VoteType::Genesis]),
+        CertificateType::FinalizeFast(_, _) => (0.8, &[VoteType::Notarize]),
+        CertificateType::Finalize(_) => (0.6, &[VoteType::Finalize]),
+        CertificateType::Skip(_) => (0.6, &[VoteType::Skip, VoteType::SkipFallback]),
+        CertificateType::Genesis(_, _) => (GENESIS_VOTE_THRESHOLD, &[VoteType::Genesis]),
     }
 }
 
 /// Lookup from `Vote` to the `CertificateId`s the vote accounts for
 ///
 /// Must be in sync with `certificate_limits_and_vote_types` and `VoteType::get_type`
-pub fn vote_to_certificate_ids(vote: &Vote) -> Vec<Certificate> {
+pub fn vote_to_certificate_ids(vote: &Vote) -> Vec<CertificateType> {
```

Comment:
We should probably rename this function, there are no certificate_ids now.

Maybe find and replace all `certificate_id` reference.